CIP-governance-metadata-actions/README.md

```diff
@@ -0,0 +1,98 @@
+
+Motivation for tooling:
+By standardizing off-chain metadata formats we facilitate interoperability for tooling which creates and/or renders metadata attached to governance actions.
+This intern promotes the user experience between tooling.
+
+The stake holders for this proposal are all of those users who intend to participate within governance via voting directly.
+Furthermore, governance related indexing and submission based tooling providers. 
+
+## Specification
+Although there are seven types of governance action defined via CIP-1694, we focus this proposal on defining core properties which must be attached to all types.
+We leave room for future standards to refine and specialize further to cater more specific for each type of governance action.
+
+### Extended Vocabulary
+The following properties extend the vocably of CIP-100.
+
+#### Title
+- This should be a sentence which should act as a human readable identifier for the GA
+- It should briefly introduce the GA
+  - i.e; Increase K protocol parameter to 100,000 to increase decentralization of Cardano.
+- The intention for this field is to act as a quick and clear differentiator between governance actions
+
+#### Abstract
+- This should be a short amount of free flow text, limited to ~200 words.
+- This should a short description of the reason behind the GA, so if a problem is being addressed, what why and how
+- This should summarizes the problem and how GA fixes the problem - to help voters gain an understanding the context behind the GA
+
+#### Motivation
+- This should be longer free form text encapsulating context around the problem that the GA is addressing.
+- The content should also include mention of the use cases and stakeholders
+- This field is an opportunity for the author to provide full context to the problem being addressed.
+
+#### Rationale
+- This should be longer free form text discussing how the GA addresses the problem identified in the motivation
+- i.e "by increasing X parameter we increase rewards for SPOs thus encouraging new SPOs to join the network
+- This section should fully justify the changes being made to Cardano
+- This should include what led to particular design decisions
+- It should describe alternate designs considered and related work.
+- The rationale should provide evidence of consensus within the community and discuss significant objections or concerns raised during the discussion.
+- For some GAs this section SHOULD to be very long.
+
+### Versioning
+This proposal should not be versioned, to update this standard a new CIP should be proposed.
+
+## Rationale: how does this CIP achieve its goals?
+- base standard, good for MVP, makes this more straight forward to implement and support for all
+- want to start simple with this proposal - MVG
+- fields based on the CIP process with the specification and most of the header removed
+
+### Open Questions
+- Should fields be optional or compulsory?
+
```

Comment:
I think we should also explore whether different kinds of governance actions require different fields, for example Treasury withdrawals will probably be more akin to Catalyst proposals than what is described here. People will also be more interested in the team behind the proposal and whether they can deliver, than with e.g. a parameter change GA.

Reply:
That is going to make this one too complex, I will leave that for other more specific CIPs

Reply:
So a CIP specifically aimed at providing a structure for GA's cant handle the complexity of structuring metadata for different types of GA? I think this is at least a topic for discussion.

Reply:
correct, this is for common properties of GA.

I would rather start with a base spec (this one) that is a manageable size, it can then expanded

Reply:
> correct, this is for common properties of GA.

Ok understood!